task_functions.py

```diff
@@ -1321,3 +1322,14 @@ def put_object_minio(
             object_name=minio_path,
             file_path=local_path + filename,
         )
+
+
+def flush_cache(host, port, db, password):
+    redis_client = redis.Redis(
+        host=host,
+        port=port,
+        db=db,
+        password=password,
+    )
+    # Delete keys in the background in a different thread without blocking the server
+    redis_client.execute_command("FLUSHALL ASYNC")
```

Comment:
This will flush cache in async - I suppose - so you would not know for sure if cache has been really flush after that